subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/DependencyInsightReportTask.java

```diff
@@ -173,11 +186,32 @@ public boolean isShowSinglePathToDependency() {
      *
      * @since 4.9
      */
-    @Option(option = "singlepath", description = "Show at most one path to each dependency")
+    @Option(option = "single-path", description = "Show at most one path to each dependency")
```

Comment:
Why change this option?

Reply:
This was changed to align the two-word options to always use a hyphen (`--all-variants`). Not sure if this should occur or not, thoughts @big-guy?

Reply:
This is a breaking change, so we can't do this without deprecations and all of that.  I wouldn't bother even though it's not consistent.